Source/MonoGame.Extended/Maps/Tiled/TiledMap.cs

```diff
@@ -20,19 +25,21 @@ public TiledMap(GraphicsDevice graphicsDevice, int width, int height, int tileWi
             _graphicsDevice = graphicsDevice;
             _layers = new List<TiledLayer>();
             _tilesets = new List<TiledTileset>();
+            Orientation = orientation;
         }
         
         private readonly List<TiledTileset> _tilesets;
         private readonly GraphicsDevice _graphicsDevice;
         private readonly List<TiledLayer> _layers;
         
-        public int Width { get; private set; }
-        public int Height { get; private set; }
-        public int TileWidth { get; private set; }
-        public int TileHeight { get; private set; }
+        public int Width { get; }
+        public int Height { get; }
+        public int TileWidth { get; }
+        public int TileHeight { get; }
```

Comment:
I haven't been using any C# 6.0 in the project yet. I want to keep it compatible with VS2013 for a while until more people are using VS2015.